src/cosalib/cmdlib.py

```diff
@@ -391,7 +391,9 @@ def import_oci_archive(parent_tmpd, ociarchive, ref):
                                         '--list', 'ostree/container/image'],
                                        encoding='utf-8').splitlines()
         assert len(refs) == 1
-        subprocess.check_call(['ostree', 'refs', '--repo', tmpd, refs[0], '--create', ref])
+        # --force is needed for builds from cosa build-oci (using buildah) where
+        # the ref seems to be already present in the repo.
+        subprocess.check_call(['ostree', 'refs', '--repo', tmpd, refs[0], '--create', ref, '--force'])
```

Comment:
I'm not sure this is needed. What we need to do is make our versions unique on every build. Right now it's just [statically set](https://github.com/coreos/fedora-coreos-config/blob/5603f9a31091b4ddef4eb3368ada64f88f2b05c4/build-args.conf#L5).

So yeah. If you do consecutive container builds they are going to have conflicts, but we should make our tooling not have that problem.